.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+# \<Description\>
+
+Attach a link to issue if relevant
+
+## What
+
+Specify what you're trying to achieve
+
+## Why
+
+Specify why you need to achieve this
+
+## Screenshots / Gifs
+
+Attach Screenshots / Gifs to help reviewers understand the scope of the pull request
+
+## Checklist:
+
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Linting and formatting pass locally with my changes
```

Comment:
I think we should just trust the CI which checks three of these. For some changes I don't bother running all tests, linting, formatting, or builds locally when I can have the CI do it for me.

Reply:
Yeah my main wonder was that the review is dismissed when changes are added. In this case if the CI fails on lint/format the reviewer needs to review again (which is fine as long as they don't amend the changes to the last commit and force push).

Reply:
Same though for me, can't be bothered to run the tests and linting for every PR. But formatting is a pretty quick task.
Even for this small PR, it failed first due to formatting.

Reply:
Right. Let's keep a check about formatting, that's probably the most common fail. We can always add more later.